CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Fixed`
 
+- [#32](https://github.com/nf-core/detaxizer/issues/32) - Addition of quotation marks in `parse_kraken2report.nf` prevents failure of the pipeline when using a taxon with space (e.g. Homo sapiens) with the `tax2filter` parameter.
```

Comment:
Usually I link PRs.

Reply:
Makes more sense to link PRs, I think. Thanks for the hint!